### ISSUE TITLE
Can't install Ubunto on an old BQ4.5 Ubunto smartphone

### ISSUE THREAD

**Issue by msantoscosta** [ubports/ubports-installer#758]
*Automatically generated error report* 
UBports Installer Version: 0.2.2-beta 
Device: krillin
Channel: Not yet set
Package: snap
Operating System: Ubuntu Linux  18.04  bionic x64 
NodeJS version: v8.2.1 

Error log: http://paste.ubuntu.com/p/WhW2KZktdP/ 

When I connect my BQ4.5 Unbunto Version to my PC with Ubunto, UBports recognize the phone but stalls at the beginning and it do nothing.

**Comment by msantoscosta** [comment]
Good afternoon,

Thank you for your reply.

Yes. I pugged my smartphone to my PC and run the UBPorts. This program recognizes my smartphone, BQ4.5, but stop´s, it does nothing.

I did restart the program several times, always with the same result.

My point is that, my smartphone has the same basic functions like the day I brought it. It doesn´t connect to the Ubunto Store (it was discontinued), so I can´t install new apps. I would like to instal a "Terminal" app.

Regards,
